test/python/circuit/test_tools.py

```diff
@@ -31,10 +31,36 @@ class TestPiCheck(QiskitTestCase):
                    (2.999999999999999, '3'),
                    (0.99, '0.99'),
                    (0.999999999999999, '1'),
+                   (pi, 'pi'),
+                   (-pi, '-pi'),
+                   (3*pi, '3pi'),
+                   (-3*pi, '-3pi'),
+                   (pi/35, 'pi/35'),
+                   (-pi/35, '-pi/35'),
+                   (3*pi/35, '0.26928'),
+                   (-3*pi/35, '-0.26928'),
+                   (pi**2, 'pi**2'),
+                   (-pi**2, '-pi**2'),
+                   (1e9, '1e+09'),
+                   (-1e9, '-1e+09'),
+                   (1e-9, '1e-09'),
+                   (-1e-9, '-1e-09'),
+                   (6*pi/11, '6pi/11'),
+                   (-6*pi/11, '-6pi/11'),
+                   (6*pi/1, '6pi'),
+                   (-6*pi/1, '-6pi'),
+                   (6*pi/2, '3pi'),
+                   (-6*pi/2, '-3pi'),
+                   (1j*3/(7*pi), '3/7pij'),
+                   (-1j*3/(7*pi), '-3/7pij'),
                    (6*pi/5+1j*3*pi/7, '6pi/5+3pi/7j'),
                    (-6*pi/5+1j*3*pi/7, '-6pi/5+3pi/7j'),
                    (6*pi/5-1j*3*pi/7, '6pi/5-3pi/7j'),
                    (-6*pi/5-1j*3*pi/7, '-6pi/5-3pi/7j'),
+                   (1/pi, '1/pi'),
+                   (-1/pi, '-1/pi'),
+                   (6/(5*pi), '6/5pi'),
+                   (-6/(5*pi), '-6/5pi'),
                    (-382578.0+.0234567j, '-3.8258e+05+0.023457j'),
                    (-382578.0-.0234567j, '-3.8258e+05-0.023457j'),
```

Comment:
We should maybe extend this list with `Parameters`.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/4807/commits/f500de55b072b0438cd139a72f411a0acc263d1e.